llvm/test/Transforms/LoopVectorize/AArch64/vector-libcall-linear-args.ll

```diff
@@ -0,0 +1,275 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter "call.*(modf|modff|sincos|sincosf|sincospi|sincospif)" --version 4
```

Comment:
I was rather aiming to keep the tests in :
llvm/test/Transforms/LoopVectorize/AArch64/sleef-calls-aarch64.ll
llvm/test/Transforms/LoopVectorize/AArch64/armpl-calls.ll

so we have a single file per library, what do you think?

Reply:
The mentioned test files have been refactored in #76146.